code/game/machinery/newscaster.dm

```diff
@@ -56,6 +58,27 @@
 	backup_author = ""
 	img = null
 	backup_img = null
+	img_pda = null
+	backup_img_pda = null
+
+/datum/feed_message/proc/make_PDA_img()
+	if (img)
+		img_pda = icon(img)
+		//turns the image grayscale then applies an olive coat of paint
+		img_pda.MapColors(rgb(77,77,77), rgb(150,150,150), rgb(28,28,28), rgb(128,128,0))
+		//lowers the brightness then ups the contrast so we get something that fits on a PDA screen
+		img_pda.MapColors(1,0,0,0,0,1,0,0,0,0,1,0,0,0,0,1,-0.53,-0.53,-0.53,0)
+		img_pda.MapColors(1.75,0,0,0,0,1.75,0,0,0,0,1.75,0,0,0,0,1.75,-0.375,-0.375,-0.375,0)
+
+/datum/feed_message/proc/NewspaperCopy()//We only copy the vars we'll need
```

Comment:
The inconsistence between this proc and the previous one really bothers me

Reply:
you mean CamelCase and underscore? or the actual naming?

Reply:
Changed make_PDA_image to ImagePDA